Rust/Rust.sublime-syntax

```diff
@@ -238,9 +238,10 @@ contexts:
         1: punctuation.definition.annotation.rust
         2: punctuation.section.group.begin.rust
       push:
+        - meta_scope: meta.annotation.rust
```

Comment:
Not quite sure about this, but without it the string doesn't get the `meta.annotation.rust` scope.